quality/finder/finder_standard.inst.cfg

```diff
@@ -11,45 +11,47 @@ weight = 0
 global_quality = True
 
 [values]
-adhesion_type = skirt
 layer_height = 0.18
 layer_height_0 = 0.27
+
+top_layers = 7
+bottom_layers = 3
+adhesion_type = skirt
 speed_slowdown_layers = 1
 travel_retract_before_outer_wall = True
-bottom_layers = 3
 cool_fan_full_layer = 1
 cool_min_layer_time = 8
 cool_min_speed = 15
-default_material_print_temperature = 210
 expand_skins_expand_distance = 0.1
 infill_enable_travel_optimization = True
 infill_pattern = zigzag
-line_width = 0.48
-material_final_print_temperature = 210
 material_flow = 109
-material_initial_print_temperature = 210
 meshfix_maximum_resolution = 0.05
 optimize_wall_printing_order = True
 retract_at_layer_change = True
 retraction_amount = 1.3
 retraction_combing_max_distance = 2
 retraction_speed = 30
 skin_preshrink = 0.1
-speed_print = 50
-speed_travel = 100
+speed_print = 80
+speed_travel = 120
 speed_wall_x = 35
 support_bottom_enable = False
 support_infill_rate = 10
 support_interface_density = 50
 support_interface_enable = True
 support_interface_skip_height = 0.2
 support_line_distance = 4.5
-support_z_distance = 0.18
-top_layers = 7
+support_z_distance = =layer_height 
```

Comment:
nice, but did you print support test prints and checked it?

Reply:
Good catch. Do you know of a good test file for testing supports? I'll print a "cube in the air" so that it generates the supports as a first try.